packages/browser/src/index.ts

```diff
@@ -1,4 +1,24 @@
-import { OidcConfigResponse, Requester } from '@logto/js';
+import {
+  fetchOidcConfig,
+  generateCodeChallenge,
+  generateCodeVerifier,
+  generateSignInUri,
+  generateState,
+  OidcConfigResponse,
+  Requester,
+  withReservedScopes,
+} from '@logto/js';
+
+const discoveryPath = '/oidc/.well-known/openid-configuration';
+const logtoStorageItemKeyPrefix = `logto:`;
+
+export const getLogtoKey = (key: string) => `${logtoStorageItemKeyPrefix}${key}`;
```

Comment:
would ${prefix}:${key} is cleaner?  more informable that colon is used to slipt the key and prefix?